### PR TITLE
add domain_code fallback

### PR DESCRIPTION
**Pull request by T0biii**
add domain_code "ffmuc_uisp_respondd_fallback" 
there is no domain, or?

### CODE REVIEW

**Review by GoliathLabs — APPROVED** [review]

**Comment by grische** [comment]
@T0biii I think this PR is not sufficient for it to work. I added a new PR #3 that should work.

File: uisp_respondd/uisp_client.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ class Accesspoint:
     latitude: float
     longitude: float
     neighbour: str
+    domain_code: str
```
Comment (grische) [inline uisp_respondd/uisp_client.py:31]: AFAICT there some changes in respondd_client.py missing.

**Review by grische — CHANGES_REQUESTED** [review]
@T0biii I think this PR is not sufficient for it to work. I added a new PR https://github.com/freifunkMUC/uisp_respondd/pull/3 that addresses these as I had no push rights on your repo

**Comment by T0biii** [comment]
see #3

**Pull request closed without merging**